src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -2,25 +2,34 @@
 
 import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 
-import seedu.address.commons.core.index.Index;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
 import seedu.address.logic.commands.DeleteCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Nric;
+import seedu.address.model.person.NricMatchesPredicate;
+
+
 
 /**
  * Parses input arguments and creates a new DeleteCommand object
  */
 public class DeleteCommandParser implements Parser<DeleteCommand> {
-
+    private static final Logger logger = LogsCenter.getLogger(DeleteCommandParser.class);
     /**
      * Parses the given {@code String} of arguments in the context of the DeleteCommand
      * and returns a DeleteCommand object for execution.
+     *
      * @throws ParseException if the user input does not conform the expected format
```

Comment:
Missing punctuation for @throws. Could perhaps be given command but might be best to make edits?

Reference: https://se-education.org/guides/conventions/java/intermediate.html